tests/php/FilterExpressionParsing/ParseResultTest.php

```diff
@@ -7,11 +7,14 @@
 use WMDE\VueJsTemplating\FilterExpressionParsing\ParseResult;
 use WMDE\VueJsTemplating\JsParsing\BasicJsExpressionParser;
 
+/**
+ * @covers \WMDE\VueJsTemplating\FilterExpressionParsing\ParseResult
+ */
 class ParseResultTest extends TestCase {
 
 	private $defaultFilters;
 
-	protected function setUp() {
+	protected function setUp(): void {
```

Comment:
Not sure what the code style within your team is. Just wanted to note that the MediaWiki style is to use spaces on both sides of the colon, e.g. `setUp() : void`.

Reply:
Wikibase generally skips the first space. You even argued for this style yourself :) https://www.mediawiki.org/wiki/Manual_talk:Coding_conventions/PHP#Declaring_of_return_types